File: TINHTONG1.py
'''
Input: 
1 số nguyên n
ouput: 
kết quả của phép toán: 1+1/23+1/33+...+1/n3 
(kết quả lấy 3 chữ số thập phân)

ví dụ:
input:
3

output:
1.162

'''
n = int(input())
kq = 0
for i in range(1, n + 1):
    kq += 1 / (i ** 3)
print(f"{kq:.3f}")

File: LTCB/C++/VongLap/Revise/TINHTONG1.cpp
/*
Input: 
1 số nguyên n
ouput: 
kết quả của phép toán: 1+1/2^3+1/3^3+...+1/n^3 
(kết quả lấy 3 chữ số thập phân)

ví dụ:
input:
3

output:
1.162
*/
#include <iostream>
#include <iomanip>
using namespace std;

int main(){
    cout << fixed << setprecision(3);
    int n;
    cin >> n;
    double tong = 0.0;
    for(int i = 1; i <= n; i++){
        tong += 1.0 / (i *i *i);
    }
    cout << tong + 1.0;
    return 0;
}